app/helpers/users_helper.rb

```diff
@@ -0,0 +1,20 @@
+module UsersHelper
+
+  def owned_books(user)
+    user.books
+  end
+
+  def reserved_books(user)
+    reserved_or_borrowed(user).select { |b| b.status == Book::RESERVED }
```

Comment:
we could create a `reserved` and `borrowed` scope on the Book model

Reply:
Ah, just what I needed! I've added the scopes and refactored UsersHelper's methods. 😀